resource-managers/kubernetes/docker/src/main/dockerfiles/spark/Dockerfile

```diff
@@ -47,7 +57,8 @@ COPY bin /opt/spark/bin
 COPY sbin /opt/spark/sbin
 COPY kubernetes/dockerfiles/spark/entrypoint.sh /opt/
 COPY kubernetes/dockerfiles/spark/decom.sh /opt/
-COPY examples /opt/spark/examples
+## Lacework modification : skip the examples to keep image lighter.
+#COPY examples /opt/spark/examples
 COPY kubernetes/tests /opt/spark/tests
```

Comment:
I'm guessing we don't need a directory named `tests` in our image?

Reply:
Agreed, I will remove it.